docker/jupyter_notebook_config.py

```diff
@@ -0,0 +1,17 @@
+# pulled from https://hub.docker.com/r/ansingh7115/leonardo-notebooks/
+# and added base_url configs for local proxy testing
+
+from jupyter_core.paths import jupyter_data_dir
+import subprocess
+import os
+import errno
+import stat
+
+c = get_config()
+c.NotebookApp.ip = '*'
+c.NotebookApp.port = 8000
+c.NotebookApp.open_browser = False
+c.NotebookApp.base_url = '/notebooks/test/'
```

Comment:
So, I think for the proxy to work correctly, we'll want to set `c.NotebookApp.base_url` and `c.NotebookApp.webapp_settings` for real notebook servers as well. Hopefully that will be possible through the init script.